doc/Developers-Intro.md

```diff
@@ -42,29 +42,64 @@ If you have seen Friendica you probably have ideas to improve it, haven't you?
 * Make plans for a better Friendica interface design and share them with us.
 * Tell us if you are able to realize your ideas or what kind of help you need.
 We can't promise we have the right skills in the group but we'll try.
-* Choose a thing to start with, e.g. work on the icon set of your favourite theme
+* Choose a thing to start with, e.g. work on the icon set of your favorite theme
 
 Programming
 ---
 
+###Coding standards
+
+For the sake of consistency between contribution and general code readability, Friendica follows the widespread [PSR-2 coding standards](http://www.php-fig.org/psr/psr-2/) to the exception of a few rules.
+Here's a few primers if you are new to Friendica or to the PSR-2 coding standards:
+ * Indentation is tabs, period (not PSR-2).
+ * By default, strings are enclosed in single quotes, but feel free to use double quotes if it makes more sense (SQL queries, adding tabs and line feeds).
+ * Operators are wrapped by spaces, e.g. `$var === true`, `$var = 1 + 2` and `'string' . $concat . 'enation'`
+ * Braces are mandatory in conditions
+ * No closing PHP tag
+ * No trailing spaces
+
+Don't worry, you don't have to know by heart the PSR-2 coding standards to start contributing to Friendica.
+There are a few tools you can use to check or fix your files before you commit.
+
```

Comment:
One thing that crossed my mind--even though not strictly coding standard--could you add the "one line per sentence" rule for the docs below the actual coding standards?